appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -434,12 +443,37 @@
                          registeredComponentName eventName)
                        #f))))
 
+       ;; For the HandlesEventDispatching interface
+       (define (dispatchMultiEvent eventName :: java.lang.String
```

Comment:
I was wondering if you could help me debug this function. I'm not really sure how to even begin to do this for this file and I'm not really knowledge about this language at all. 

Basically I have multi-events registered separately, and whenever a normal event is called, the correct multi-event will also be dispatched if it exists (currently hardcoded only for buttons right now). Currently, an app using the multi-event block fails when the event is dispatched. If in the EventDispatcher, if I comment out the line "dispatchDelegate.dispatchMultiEvent(multiEventName, newArgs);" then no error occurs.